samples/cpp/videocapture_basic.cpp

```diff
@@ -1,54 +0,0 @@
-/**
-  @file videocapture_basic.cpp
```

Comment:
I propose to not touch VideoCapture related samples. We have dedicated PR for it.